.github/workflows/people.yml

```diff
@@ -58,6 +58,35 @@ jobs:
             exit 1
           fi
 
+  build-front-desk:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: src/frontend/app/desk
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+      - name: Install yarn
+        run: npm install -g yarn
+      - name: Restore cache
+        uses: actions/cache@v3
```

Comment:
For a future PR : would it be beneficial to restore cache before doing the install?

Reply:
The first install is to install `yarn`, the dependency install is after the cache restoration https://github.com/numerique-gouv/people/blob/c0969b410baa97ae31975501b28198613206977a/.github/workflows/people.yml#L86

But apparently, `actions/setup-node@v4` has already a caching system, so I will use it, it will simplify the job: https://github.com/actions/setup-node?tab=readme-ov-file#caching-global-packages-data